dictionary.lisp

```diff
@@ -0,0 +1,120 @@
+#|
+ This file is a part of Qtools-UI
+ (c) 2019 Shirakumo http://tymoon.eu (shinmera@tymoon.eu)
+ Author: Michał "phoe" Herda <phoe@disroot.org>
+|#
+
+(in-package #:org.shirakumo.qtools.ui)
+(in-readtable :qtools)
+
+(define-widget dictionary (qwidget)
+  ((empty-browser-text :accessor empty-browser-text
+                       :initarg :empty-browser-text)
+   (not-found-text :accessor not-found-text
+                   :initarg :not-found-text))
+  (:default-initargs
+    :empty-browser-text
+    "<i><p align=center>Type your query below and hit Search.</p></i>"
+    :not-found-text
+    "<i><p align=center>The entry for \"~A\" was not found.</p><i>"))
+
+(define-subwidget (dictionary layout) (q+:make-qgridlayout)
```

Comment:
If you `(q+:make-whateverlayout my-widget)` you don't need to `(setf (q+:layout ..) ..)` after. It's cleaner and shorter.

Reply:
Fixed - TIL. I was not aware of this.

Reply:
You don't need to comment on each thing as fixed. If you push the patches it'll notice that it's changed automatically.

Reply:
Woooo. I keep on learning GitHub stuff every day.